### PR TITLE
add cheat to remove station loading penalty

### PR DESCRIPTION
**Pull request by LeftofZen**
Add a cheat to disable slow loading at stations when vehicle length exceeds station length.

### CODE REVIEW

File: src/OpenLoco/src/Vehicles/VehicleHead.cpp [hunk 1]
```diff
@@ -2552,6 +2552,11 @@ namespace OpenLoco::Vehicles
 
     uint8_t VehicleHead::getLoadingModifier(const VehicleBogie* bogie)
     {
+        if (Config::get().disableVehicleLoadPenaltyCheat)
+        {
+            return 1;
+        }
+
```
Comment (duncanspumpkin) [inline src/OpenLoco/src/Vehicles/VehicleHead.cpp:2559]: L2598 is logic that should probably still occur. I origianlly wanted to make this a nice pure function when i yanked it out of where it was being used but i didn't want to duplicate the logic for L2598.

Comment (LeftofZen) [reply src/OpenLoco/src/Vehicles/VehicleHead.cpp:2556]: oh yeah good catch, fixed

File: src/OpenLoco/src/Vehicles/VehicleHead.cpp [hunk 2]
```diff
@@ -2552,18 +2552,20 @@ namespace OpenLoco::Vehicles
 
     uint8_t VehicleHead::getLoadingModifier(const VehicleBogie* bogie)
     {
+        constexpr uint8_t minVehicleLoadingModifier = 1;
```
[the remark this answers is not in the document]
Comment (duncanspumpkin) [reply src/OpenLoco/src/Vehicles/VehicleHead.cpp:2555]: Should probably also just have a kTrackPastStationPenalty and kRoadPastStationPenalty

**Review by duncanspumpkin — APPROVED** [review]

File: src/OpenLoco/src/Vehicles/VehicleHead.cpp (hunk 2 above)
[the remark this answers is not in the document]
Comment (LeftofZen) [reply src/OpenLoco/src/Vehicles/VehicleHead.cpp:2555]: good call, I will update that

[the remark this answers is not in the document]
Comment (LeftofZen) [reply src/OpenLoco/src/Vehicles/VehicleHead.cpp]: done

**Pull request merged**